store/postgres/Cargo.toml

```diff
@@ -8,10 +8,14 @@ diesel = { version = "1.3.2", features = ["postgres", "serde_json", "numeric"] }
 diesel_migrations = "1.3.0"
 ethereum-types = "0.3"
 diesel-dynamic-schema = { git = "https://github.com/diesel-rs/diesel-dynamic-schema" }
+fallible-iterator = "0.1.4"
 futures = "0.1.21"
 graph = { path = "../../graph" }
-slog = "2.2.3"
+postgres = "0.15.2"
+serde = "1.0"
 serde_json = "1.0"
+slog = { version = "2.2.3", features = ["release_max_level_debug"] }
```

Comment:
`slog` should only be in the `Cargo.toml` of the `graph` crate, where we control the logging level. To import the slog stuff you need to `use graph::prelude::*`.